pages/maps/_id/edit.vue

```diff
@@ -0,0 +1,85 @@
+<template>
+  <div class=" justify-center items-center flex flex-grow">
+    <div class="bg-blue-900 p-5 rounded-sm main-container">
+      <form @submit.prevent="onSubmit" class="flex flex-col">
+        <h1 class="text-white text-xl mb-2">Edit Map</h1>
+        <!-- <p v-if="errors.length" class="text-red-300 bg-red-800 p-3 mb-3 rounded-sm">
+          <strong>Please correct the following error(s):</strong>
+          <ul>
+            <li v-for="error in errors">{{ error }}</li>
+          </ul>
+        </p> -->
+        <input
+          type="text"
+          name="name"
+          v-model="name"
+          class="mb-4 p-2 rounded-sm"
+          placeholder="Map Name"
+        />
+        <input
+          type="text"
+          name="name"
+          v-model="imageHeight"
+          class="mb-4 p-2 rounded-sm"
+          placeholder="Image Height"
+        />
+        <input
+          type="text"
+          name="name"
+          v-model="imageWidth"
+          class="mb-4 p-2 rounded-sm"
+          placeholder="Image Width"
+        />
+        <div class="mb-3">
+          <img :src="previewImage" class="preview mb-3 max-w-xs" />
+          <input type="file" accept="image/*" @change="onImageChange" />
+        </div>
+        <!-- <input type="submit" v-bind:value="Create Map" class="btn btn-primary"  /> -->
+        <button type="submit" class="btn btn-primary" :disabled="submitting">
+          Save Map
+        </button>
+      </form>
+      <p class="text-center mt-2">
+        <nuxt-link to="/" class="text-white">Cancel</nuxt-link>
```

Comment:
Nitpicky, but I'm transitioning to CamelCase app names. I'm pretty sure that's the new vue3 spec.

`<NuxtLink />`

Reply:
https://router.vuejs.org/api/#to

`NuxtLink` is just fancy `vue router-link`

Try using route names instead., or full objects like `<NuxtLink :to='{ name: "edit-map", params: { id: 69 } }' />

Reply:
how do you use route names? this got cut off i think

Reply:
updated comment

Reply:
Oh I also forgot to mention.

Nuxt does dynamic route names, I don't use them personally.
https://nuxtjs.org/guide/routing/#basic-routes


I use a custom router.js + hand craft each route+name.
https://github.com/nuxt-community/router-module